.circleci/config.yml

```diff
@@ -353,17 +353,15 @@ jobs:
               echo "Not a PR; skipping"
             fi
       - run:
-          name: Setup registry config for using package previews on draft PRs
+          name: Install dependencies
           command: |
             if [[ $IS_DRAFT == 'true' ]]
             then
-              printf '%s\n\n%s' '@metamask:registry=https://npm.pkg.github.com' "//npm.pkg.github.com/:_authToken=${GITHUB_PACKAGE_READ_TOKEN}" > .npmrc
+              # Use GitHub registry on draft PRs, allowing the use of preview builds
+              METAMASK_NPM_REGISTRY=https://npm.pkg.github.com yarn --immutable
```

Comment:
I first tried setting this environment variable in a separate step (similar to how `IS_DRAFT` is set), but it didn't work for reasons I wasn't able to discern. Combing the steps seems to work fine though